mobile/src/main/java/ch/epfl/sdp/mobile/application/Profile.kt

```diff
@@ -1,5 +1,7 @@
 package ch.epfl.sdp.mobile.application
 
+import androidx.compose.ui.graphics.Color
```

Comment:
MVVM violation : you should not use `androidx.compose.ui.graphics.Color` in the `application` package.

Reply:
it is removed